cluster-up/cluster/k8s-1.17/manifests/ovsdpdk/allinone.yaml

```diff
@@ -0,0 +1,310 @@
+- apiGroups: ["ovsdpdknetwork.openshift.io"]
+  resources: ["*"]
+  verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ovsdpdk-network-prepare
+rules:
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch", "patch", "update"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["*"]
+- apiGroups: ["apps"]
+  resources: ["daemonsets"]
+  verbs: ["get"]
+- apiGroups: [""]
+  resources: ["pods/eviction"]
+  verbs: ["create"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ovsdpdk-network-operator
+roleRef:
+  kind: ClusterRole
+  name: ovsdpdk-network-operator
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  namespace: default
+  name: ovsdpdk-network-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: ovsdpdk-network-prepare
+roleRef:
+  kind: ClusterRole
+  name: ovsdpdk-network-prepare
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  namespace: default 
+  name: ovsdpdk-network-prepare
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  creationTimestamp: null
+  name: ovsdpdk-network-operator
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - services/finalizers
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - get
+  - create
+- apiGroups:
+  - apps
+  resourceNames:
+  - ovsdpdk-network-operator
+  resources:
+  - deployments/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - apps
+  resources:
+  - replicasets
+  - deployments
+  verbs:
+  - get
+- apiGroups:
+  - ovsdpdknetwork.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ovsdpdk-network-operator
+subjects:
+- kind: ServiceAccount
+  name: ovsdpdk-network-operator
+roleRef:
+  kind: Role
+  name: ovsdpdk-network-operator
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ovsdpdk-network-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ovsdpdk-network-operator
+  template:
+    metadata:
+      labels:
+        name: ovsdpdk-network-operator
+    spec:
+      serviceAccountName: ovsdpdk-network-operator
+      containers:
+        - name: ovsdpdk-network-operator
+          image: quay.io/krsacme/ovsdpdk-network-operator:latest
```

Comment:
You can't reference "latest" images. Please also pre-pull the image.

Reply:
Updated the ovsdpdk operator version to a specific version.